src/providers/ruby.rs

```diff
@@ -108,13 +113,12 @@ impl RubyProvider {
     }
 
     fn get_build(&self, app: &App) -> Result<Option<Phase>> {
+        let mut build = Phase::build(None);
         if self.is_rails_app(app) {
-            Ok(Some(Phase::build(Some(
-                "bundle exec rake assets:precompile".to_string(),
-            ))))
-        } else {
-            Ok(None)
+            build.add_cmd("bundle exec rake assets:precompile".to_string());
         }
+
+        Ok(Some(build))
     }
```

Comment:
With the only include Gemfile addition, `COPY . /app/` was never being run for none Rails apps causing an obvious failure.

This change means Ruby apps now always have a build phase, even if no commands are run. This ensures source files get copied.